CMakeLists.txt

```diff
@@ -78,9 +78,6 @@ add_cppcheck_dirs("${PROJECT_SOURCE_DIR}/lib" "${PROJECT_SOURCE_DIR}/exe")
 enable_cppcheck()
 
 
-# We duplicate the `help_test` as `smoke_test` until the default functionality
-# of the tool has been nailed down.
-add_test(NAME "db_smoke_test" COMMAND puppet-db)
```

Comment:
We'll have to do that at a higher level, since this command (`puppet-db`) is actually incomplete and so the output is directed to nowide::cerr and thus the cli returns a non-zero exit status.